examples/counter-react/src/store/counter.store.js

```diff
@@ -25,6 +25,6 @@ const useCounterContainer = () => {
   };
 };
 
-let [CounterContext, CounterWrapper, counterRef] = setupStore(useCounterContainer);
+let counterRef = setupStore(useCounterContainer);
 
-export { CounterContext, CounterWrapper, counterRef };
+export { counterRef };
```

Comment:
in these examples, lets make this now the default export for simplicity, and also update the name to be CounterStore (since this is now the entire store object). Then update all the code in the examples to work with these changes 😉 .